contents/blog/product-led-growth.md

```diff
@@ -0,0 +1,193 @@
+
+Facebook and Slack are great examples of this. 
+
+### Revenue 
+Since the initial focus is on building a product rather than selling it, you will likely make less money in the short-run.
+
+With the goal not being to hit a Sales metric, you might be turning down opportunities to generate revenue if they are not perfectly aligned with what you're building.
+
+As such, you need to make sure your team and investors are aligned with this. It should be understood that it's okay to not be making money in the early stages.
+
+## One Size Doesn't Fit All
+Product-Led Growth is not a magic pill. It does have its shortcomings.
+
+Primarily, how can you let your product speak for itself if you can't get anyone to look at it in the first place?
+
+But rather than tell you when Product-led Growth might not work, I'll tell you what aspects can be beneficial to it.
+
+### Funding
+It's much easier to focus almost exclusively on building a product when you're already well-funded. This is our case at PostHog.
+
+To build a great product, you need a great team. And to build a great team, you need money. 
+
+Thus, if you're a company without a lot of money in the bank, you might just need to get some Sales done now, simply to fund your operations.
+
+Not having funding shouldn't discourage you from trying a PLG approach, however. It can still work, just look at GitLab. They first raised money after already having 100,000 users and Sid (their CEO), claims they didn't even have a significant marketing budget until then. Wow. 
+
+### Open Source Software
+Being open source is a great characteristic for companies looking to follow a PLG approach. 
+
+This is because you can get users excited not only about the usability of your product, but also how it's built.
+
+Additionally, open source software projects by default encourage community participation, since anyone is able to suggest a change, raise an issue, or even contribute code. 
+
+As such, you are more involved with your users and can gain better insight into how people feel about your product.
+
+### Diversity
+Having a team with diverse backgrounds can be extremely helpful for executing a growth strategy focused on the product. 
+
+Diversity can come in various forms. One of them is breadth of professional experience, as opposed to depth. Depth refers to extensive experience in a specific field, whereas breadth means experience across fields.
+
+Depth of experience is an essential characteristic of a world-class team. However, having some focus on breadth can also be beneficial. By having individuals able to cross role descriptions and wear many hats, it becomes easier to ensure that the product is reflected in the mentality of the various teams in an organization.
+
+At PostHog, for example, we have people with Engineering backgrounds handling sales, marketing, content creation, customer support, social media, among other non-technical roles. 
```

Comment:
This feels a bit like you're arguing we need a broad range of backgrounds yet everyone is an engineer which is the opposite. Could it be worded a bit more clearly?